client.lua

```diff
@@ -150,7 +186,7 @@ local function startTestDriveTimer(testDriveTime, prevCoords)
                     TriggerServerEvent('qb-vehicleshop:server:deleteVehicle', testDriveVeh)
                     testDriveVeh = 0
                     inTestDrive = false
-                    SetEntityCoords(cache.ped, prevCoords)
+                    SetEntityCoords(cache.ped, Config.Shops[shop]['TestDriveReturnLocation'])
```

Comment:
```suggestion
                    SetEntityCoords(cache.ped, Config.Shops[shop].TestDriveReturnLocation)
```
nit; prefer dot notation instead of string literals.

Reply:
Copy that. Done it.